api/src/wfl/server.clj

```diff
@@ -89,14 +86,16 @@
   (letfn [(do-update! [{:keys [id uuid]}]
             (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
               (let [{:keys [watchers] :as workload}
-                    (workloads/load-workload-for-id tx id)]
+                    (workloads/load-workload-for-id tx id)
+                    ;; we don't support notifying emails yet
+                    slack-watchers (filter slack/slack-channel-watcher? watchers)]
                 (log/info (format "Updating workload %s" uuid))
```

Comment:
You need to lift this log higher now, I think.

Reply:
Agree!